### PR TITLE
1.17.1 port

### PR DESCRIPTION
**Pull request by rdrpenguin04**
Currently very broken; also, PR probably isn't going to the right branch. In any case, here goes nothing.

I'm going to try to fix build issues, then start making sure there aren't any major breakages in the game; only after that will I remove draft status. If anyone wants to help, let me know!

Overall Status:
- [x] Update Gradle settings to build MC 1.17.1
- [ ] Fix immediate build breakages
- [ ] Check compatibility with 1.17 engine (run the game, ask for testing)
- [ ] Check compatibility with mods

### CODE REVIEW

**Comment by bashgardev** [comment]
Hi there @rdrpenguin04, I don't have any experience modding Minecraft, but do know python and java. How could I help or contribute?

File: build.gradle [hunk 1]
```diff
@@ -2,8 +2,8 @@ plugins {
     id 'fabric-loom' version '0.6-SNAPSHOT'
```
Comment (guygrigsby) [inline build.gradle:2]: id 'fabric-loom' version '0.8-SNAPSHOT'

**Comment by abbas-rz** [comment]
it says 1.16.2+ doesn't that mean that it will work with 1.17.1?

**Comment by leocd277** [comment]
> 
> 
> it says 1.16.2+ doesn't that mean that it will work with 1.17.1?

Tried it and doesn't :/

**Comment by Finnegan161616** [comment]
> > it says 1.16.2+ doesn't that mean that it will work with 1.17.1?
> 
> Tried it and doesn't :/

yeah, should prob say 1.16.2-1.16.5 or whatever, but let's be patient


also I'm getting as low as 1/5 of the fps in 1.17.1 than I did in 1.16.5, and the main lag causer is apparently lighting updates, so I can't wait either, dw

**Comment by PistonUser** [comment]
> it says 1.16.2+ doesn't that mean that it will work with 1.17.1?

only on singleplayer

**Pull request closed without merging**